proxies/live/apiproxy/policies/RaiseFault.403MissingNRLAppID.xml

```diff
@@ -0,0 +1,26 @@
+<RaiseFault async="false" continueOnError="false" enabled="true" name="RaiseFault.403MissingNRLAppID">
+    <FaultResponse>
+        <Set>
+            <Payload contentType="application/json">
+            {
+                "resourceType": "OperationOutcome",
+                "issue": [ {
+                    "severity": "error",
+                    "code": "required",
+                    "details": {
+                        "coding": [ {
+                            "system": "https://fhir.nhs.uk/CodeSystem/Spine-ErrorOrWarningCode",
+                            "code": "PRECONDITION_FAILED",
+                            "display": "Precondition failed"
+                        } ]
+                    },
+                    "diagnostics": "The "nrl-app-id" custom attribute for your application is missing or invalid"
```

Comment:
Put also contact onboarding team to fix or something. To let them know it's not really something wrong with their code but just onboarding issues.